src/vsr/grid.zig

```diff
@@ -1179,10 +1179,6 @@ pub fn GridType(comptime Storage: type) type {
 
             if (header.checksum != expect.checksum) return .unexpected_checksum;
 
-            if (constants.verify) {
-                assert(stdx.zeroed(block[header.size..vsr.sector_ceil(header.size)]));
-            }
```

Comment:
Should we just forcefully zero-this out then, to reduce dimensionality?

Reply:
:thinking: I think that relying even on zeroed padding is a bug, so I'd rather _not_ zero the padding, to avoid potentially concealing bugs. I guess we could zero only when `!constants.verify`... but I'd rather avoid that kind of divergence between test/non-test.

Reply:
Ok, makes sense to go with offensive programming here! Worth capturing the reasoning in a comment here though!